workspace.json

```diff
@@ -472,6 +468,33 @@
     "@fluentui/react-popover": {
       "root": "packages/react-popover",
       "projectType": "library"
+    },
+    "nx-workspace-tools": {
```

Comment:
this is showcase how task managed will look like in the future with full nx adoption. for now it's leveraged only for `tools/`

Reply:
Will we be duplicating this kind of json config for all packages listed in the `workspace.json` in the future ?

Reply:
> Will we be duplicating this kind of json config for all packages listed in the workspace.json in the future ?

we might, yes -> meaning no more package.json#scripts. It gives us various benefits over primitive npm scripts, integration with vscode plugin etc.

> spoiler alert -> nx 13 will add new features that this huge workspace json will be dissected into smaller ones per package